cmd/launch.go

```diff
@@ -253,9 +253,15 @@ func runLaunch(cmdctx *cmdctx.CmdContext) error {
 		return nil
 	}
 
-	fmt.Println("Your app is ready. Deploy with `flyctl deploy`")
+	if srcInfo.SkipDeploy && srcInfo.Docs == "" {
+		fmt.Println("Your app is ready. Deploy with `flyctl deploy`")
+	} else if srcInfo.Docs != "" {
```

Comment:
This comment was always printed because people kept running `flyctl launch` for subsequent deploys. The issue seemed to go away after telling them to deploy with `flyctl deploy` then offering to do it now with a prompt, rather than making it seem like `flyctl launch` was doing something special to deploy. I don't love that solution, but we should probably keep it unless you have a better idea.